### PR TITLE
Anthony/add ssr support

### PR DESCRIPTION
**Pull request by anth-cole**
initial decoding support for SSR messages @mfine @martin-swift

### CODE REVIEW

File: c/src/rtcm3_msm_utils.c [hunk 1]
```diff
@@ -148,15 +148,17 @@ msm_enum to_msm_type(uint16_t msg_num) {
...
 /** Convert message number into constellation enum
- *
+ * This also supports SSR messages
  * \param msg_num RTCM message number
  * \return constellation enum
  */
 constellation_t to_constellation(uint16_t msg_num) {
-  if (msg_num >= 1071 && msg_num <= 1077) {
+  if ((msg_num >= 1071 && msg_num <= 1077) ||
+     (msg_num >= 1057 && msg_num <= 1062)){
```
Comment (mfine) [inline c/src/rtcm3_msm_utils.c]: What about 1265?

File: c/src/rtcm3_msm_utils.c [hunk 2]
```diff
@@ -148,15 +148,17 @@ msm_enum to_msm_type(uint16_t msg_num) {
...
     return CONSTELLATION_GPS;
   }
-  if (msg_num >= 1081 && msg_num <= 1087) {
+  if ((msg_num >= 1081 && msg_num <= 1087) ||
+     (msg_num >= 1063 && msg_num <= 1068)) {
```
Comment (mfine) [inline c/src/rtcm3_msm_utils.c]: What about 1266?

**Comment by martin-swift** [comment]
Can someone quickly give me a bit of context here?  SSR?  State Space Representation...? And these are for getting corrections?

**Comment by anth-cole** [comment]
@martin-swift Sorry for the lack of context - yep, these are the RTCM State Space Representation messages. I'm adding support for them in the c library (currently already supported in the haskell side) such that we can use the c decoder in the orion data flow. This PR is adding the low level decoding and message definitions, there's also a gnss-converters PR that adds the conversion logic.

@mfine believe this good to go now for both GPS and GLO

**Comment by mfine** [comment]
great!

**Pull request merged**